apl-core/src/main/java/com/apollocurrency/aplwallet/apl/eth/service/EthereumWalletService.java

```diff
@@ -242,13 +244,12 @@ public String sendApproveTransaction(EthWalletKey ethWalletKey, String spenderAd
     /**
      * Send Approve Transaction
      * @param spenderAddress sender address
-     * @param value amount
      * @return tx transaction id
      */
-    private String sendApproveTransaction(Credentials credentials, String spenderAddress, BigInteger value, Long gasPrice){
+    private String sendApproveTransaction(Credentials credentials, String spenderAddress, Long gasPrice) {
         String tx = null;
         try {
-            Function function = approve(spenderAddress, value);
+            Function function = approve(spenderAddress, MAX_POS_INT);
```

Comment:
I think, that amount to approve should be passed by caller code (DexSmartContractService.deposit).

Reply:
yep, it makes sense